.travis.yml

```diff
@@ -17,7 +17,7 @@ cache:
 install: mvn de.qaware.maven:go-offline-maven-plugin:1.2.1:resolve-dependencies -Pjacoco,sonar --quiet --show-version
 # do not install to avoid dirtying the cache
 script:
-  - mvn verify -Pjacoco,sonar -Dintegration-tests=true --show-version
+  - mvn verify -Prun-code-coverage,sonarcloud-analysis -Dintegration-tests=true --show-version
```

Comment:
Please replicate this change 3 lines above.

Reply:
I would say this can't work because under `sonarcloud-analysis` profile, Jacoco report execution and sonarcloud scanner are bound to `generate-sources` phase. Let's talk about this tomorrow.

Reply:
Nice catch! Yes, there has to be a separate maven run as we have for VRP:
https://github.com/kiegroup/optaweb-vehicle-routing/blob/master/.travis.yml#L24